jenkins-jobs/pipelines/release/release-pipeline.groovy

```diff
@@ -458,9 +470,6 @@ node('release-node') {
                             it.replaceFirst('<version>.+</version>\n    </parent>', "<version>$DEVELOPMENT_VERSION</version>\n    </parent>")
                         }
                     }
-                    if(id == "operator") {
-                        sh "mvn clean install -Pk8update -DskipTests -DskipITs"
-                    }
```

Comment:
I noticed that the examples and k8 maninifest were being overwritten with nightly images again in release branches. They should instead be locked to the latest patch release for that branch. Thus the removal.